react/src/components/AgentList.tsx

```diff
@@ -0,0 +1,816 @@
+        }
+      },
+    },
+    {
+      title: t('agent.DiskPerc'),
+      key: 'disk_perc',
+      render: (value, record) => {
+        const parsedDisk =
+          JSON.parse(record?.live_stat || '{}')?.node?.disk ?? {};
+        const pctValue = parseFloat(parsedDisk.pct) || 0;
+        const pct = parseFloat(pctValue.toFixed(1));
+        const color = pct > 80 ? token.colorError : token.colorSuccess;
+        return (
+          <Flex direction="column">
+            <ResourceAvailableGageBar
+              valueLabel={pct + '%'}
+              percent={pct}
+              progressColor={color}
+            />
+            <Typography.Text>
+              {iSizeToSize(parsedDisk?.current, 'G')?.numberUnit}B /{' '}
+              {iSizeToSize(parsedDisk?.capacity, 'G')?.numberUnit}B
+            </Typography.Text>
+          </Flex>
+        );
+      },
+    },
+    {
+      title: t('general.ResourceGroup'),
+      key: 'resource_group',
+      dataIndex: 'scaling_group',
+      sorter: (a, b) => {
+        const groupA = a?.scaling_group || '';
+        const groupB = b?.scaling_group || '';
+        return groupA.localeCompare(groupB);
+      },
+    },
+    {
+      title: t('agent.Status'),
+      key: 'status',
+      dataIndex: 'status',
+      render: (value, record) => {
+        const parsedComputePlugins = JSON.parse(
+          record?.compute_plugins || '{}',
+        );
+        const parsedAvailableSlots = JSON.parse(
+          record?.available_slots || '{}',
+        );
+        return (
+          <Flex direction="column" gap="xxs" align="start">
+            <DoubleTag
+              values={[
+                { label: 'Agent' },
+                {
+                  label: record?.version,
+                  color:
+                    value === 'ALIVE'
+                      ? 'green'
+                      : value === 'TERMINATED'
+                        ? 'red'
+                        : 'blue',
+                },
+              ]}
+            />
+            {parsedComputePlugins?.cuda ? (
+              <>
+                {parsedComputePlugins?.cuda?.cuda_version ? (
+                  <DoubleTag
+                    values={[
+                      { label: 'CUDA' },
+                      {
+                        label: parsedComputePlugins?.cuda?.cuda_version,
+                        color: 'green',
+                      },
+                    ]}
+                  />
+                ) : (
+                  <Tag color="green">CUDA Disabled</Tag>
+                )}
+                <DoubleTag
+                  values={[
+                    { label: 'CUDA Plugin' },
+                    {
+                      label: parsedComputePlugins?.cuda?.version,
+                      color: 'blue',
+                    },
+                  ]}
+                />
+                {_.includes(_.keys(parsedAvailableSlots), 'cuda?.shares') ? (
+                  <Tag color="blue">Fractional GPU™</Tag>
```

Comment:
This is typo, right?
```suggestion
                {_.includes(_.keys(parsedAvailableSlots), 'cuda.shares') ? (
```